packages/poller/src/invoice/processInvoices.ts

```diff
@@ -43,35 +44,52 @@ export async function processInvoices({
   config,
   web3Signer,
 }: ProcessInvoicesParams): Promise<void> {
+  // Create request id tracker for polling
   const requestId = hexlify(randomBytes(32));
+
+  // Get current time to measure invoice age
+  const time = getTimeSeconds();
+  let start = time;
+  logger.debug('Method start', { requestId, start });
+
   logger.info('Starting invoice processing', {
     requestId,
     invoiceCount: invoices.length,
     invoices: invoices.map((i) => i.intent_id),
   });
 
-  // Get current time to measure invoice age
-  const time = Math.floor(Date.now() / 1000);
-
   // Query all of marks balances across chains
   logger.info('Getting mark balances', { requestId, chains: Object.keys(config.chains) });
   const balances = await getMarkBalances(config, prometheus);
-  logBalanceThresholds(balances, config, logger);
-  logger.debug('Retrieved balances', { requestId, balances: jsonifyMap(balances) });
+  logger.debug('Retrieved balances', { requestId, balances: jsonifyMap(balances), duration: getTimeSeconds() - start });
```

Comment:
Don't want to record mark balances here anymore?

Reply:
yeah nobody is using that alerting, so why add the monitoring log?